optimum_benchmark/backends/vllm/backend.py

```diff
@@ -124,7 +124,6 @@ def batch_offline_engine_generate(self, inputs: Dict[str, Any], kwargs: Dict[str
                     n=kwargs.get("num_return_sequences"),
                     max_tokens=kwargs.get("max_new_tokens"),
                     min_tokens=kwargs.get("min_new_tokens"),
-                    use_beam_search=kwargs.get("num_beams") > 1,
```

Comment:
use_beam_search is not a parameter of samplingParams https://github.com/vllm-project/vllm/blob/e9d517f27673ec8736c026f2311d3c250d5f9061/vllm/sampling_params.py#L105

Reply:
yeh it was removed in the latest version https://github.com/vllm-project/vllm/releases/tag/v0.6.2